[macOS M1]Undefined symbols for architecture arm64

在我的 MBP M1 编译 examples/0002.a+b，发生错误：

```sh
$ g++ -Ofast -std=c++20 -flto -I../../include -o a+b a+b.cc
undef: __srget
Undefined symbols for architecture arm64:
  "__srget", referenced from:
      bool fast_io::details::scan_context_status_impl<fast_io::basic_c_family_io_observer<(fast_io::c_family)1, char>, fast_io::manipulators::scalar_manip_t<fast_io::manipulators::scalar_flags{36ul, false, false, false, false, false, false, false, true, false, false, (fast_io::manipulators::scalar_placement)0, (fast_io::manipulators::floating_format)1}, unsigned long&> >(fast_io::basic_c_family_io_observer<(fast_io::c_family)1, char>, fast_io::manipulators::scalar_manip_t<fast_io::manipulators::scalar_flags{36ul, false, false, false, false, false, false, false, true, false, false, (fast_io::manipulators::scalar_placement)0, (fast_io::manipulators::floating_format)1}, unsigned long&>) in cc-fd9180.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

然后 examples/0003.manipulators 也出错：

```sh
$ g++ -Ofast -std=c++20 -flto -I../../include -o width width.cc
width.cc:6:10: error: use of undeclared identifier 'left_width'
        println(left_width("hello",20));
                ^
width.cc:7:10: error: use of undeclared identifier 'middle_width'
        println(middle_width("hello",20));
                ^
width.cc:8:10: error: use of undeclared identifier 'right_width'
        println(right_width("hello",20));
                ^
3 errors generated.
```

--- a/examples/0003.manipulators/width.cc
+++ b/examples/0003.manipulators/width.cc
@@ -3,7 +3,7 @@
 int main()
 {
 	using namespace fast_io::mnp;
-	println(left_width("hello",20));
-	println(middle_width("hello",20));
-	println(right_width("hello",20));
+	println(left("hello",20));
+	println(middle("hello",20));
+	println(right("hello",20));
 }

